ui/src/ducks/login.js

```diff
@@ -7,11 +9,13 @@ const AUTHENTICATE = 'AUTHENTICATE';
 const AUTHENTICATION_SUCCESS = 'AUTHENTICATION_SUCCESS';
 const AUTHENTICATION_FAILED = 'AUTHENTICATION_FAILED';
 const LOGOUT = 'LOGOUT';
+const UPDATE_API_COREV1 = 'UPDATE_API_COREV1';
 
 // Reducer
 const defaultState = {
   user: null,
-  error: null
+  error: null,
+  api: null
```

Comment:
not only. api configuration depends on the url endpoint and the user token.
-const config = new Config('https://localhost:8080', token, 'Basic');
-const coreV1 = config.makeApiClient(Core_v1Api);
In the common way, I would store only Config in the redux store and pass it as parameters for API calls. I mean we have to duplicate "config.makeApiClient(Core_v1Api);" for every API call...

Reply:
Why not just store the token in Redux? I know it means re-creating a "client" object for each call, but I don't think the computational burden would be too high compared to the HTTP request RTT. And having an active token in the `login` state makes a whole lot more sense to me, don't you agree?

Reply:
The token is already in Redux :) 
These changes is to avoid re-creating a "client" object for each call.

Reply:
@NicolasT what do you think?

Reply:
Not a big fan of re-creating such 'client' objects and API wrappers on every invocation, TBH. Maybe this stuff doesn't fit in the Redux model/store/whatnot, but then that's a limitation of the design IMHO. Yes, compared to performing a query, the overhead (at least time-wise) is minimal, however it's bad form to re-create a (basically) constant object time and time again. If it can't go in the Redux store, then maybe it should go somewhere else with a (serializable) reference to the actual object in the store.

Reply:
OK I'll revert these changes and create a debt ticket for it.